src/components/course/tests/CourseSidebarPrice.test.jsx

```diff
@@ -73,90 +77,122 @@ const courseStateWithNoOffersNoLicenseSubsidy = {
   catalog: { catalogList: ['bears'] },
 };
 
+const defaultSubsidyRequestsState = {
+  subsidyRequestConfiguration: null,
+  licenseRequests: [],
+  couponCodeRequests: [],
+  isLoading: false,
+};
+
 // eslint-disable-next-line react/prop-types
 const SidebarWithContext = ({
   initialAppState = appStateWithOrigPriceHidden,
+  subsidyRequestsState = defaultSubsidyRequestsState,
   initialCourseState,
 }) => (
   <AppContext.Provider value={initialAppState}>
-    <CourseContextProvider initialState={initialCourseState}>
-      <CourseSidebarPrice />
-    </CourseContextProvider>
+    <SubsidyRequestsContext.Provider value={subsidyRequestsState}>
+      <CourseContextProvider initialState={initialCourseState}>
+        <CourseSidebarPrice />
+      </CourseContextProvider>
+    </SubsidyRequestsContext.Provider>
   </AppContext.Provider>
 );
 
 SidebarWithContext.propTypes = {
```

Comment:
curious do we need proptypes here if we disabled eslint check above?

Reply:
Weirdly enough, even though we have that `// eslint-disable-next-line react/prop-types` above, linter still yells at me if I take off propTypes. I'll keep playing around just go with one or the other.